prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -226,7 +226,7 @@ class GenerateDocs : CliktCommand(
         }
         return Schema(
             fromSchema.baseName,
-            "covid-19",
+            fromSchema.topic,
```

Comment:
This is a functional change. @carlosfelix2 are you familiar enough with this part of the CLI to confirm this will have no adverse effects?

Reply:
The comment was hidden since it is outdated, but this functional change was in response to Mo's comment [here](https://github.com/CDCgov/prime-reportstream/pull/7384#discussion_r1026503480), so I think we should be okay, but Carlos, still feel free to confirm that it's alright.

Reply:
Just saw Mo's comment. Looks good.